tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -2460,6 +2460,63 @@ public:
    using Detail::RFieldBase::GenerateValue;
 };
 
+class RNTupleBLOB {
+private:
+   std::shared_ptr<void> fData;
```

Comment:
Why `<void>` rather than `<char>`?

Reply:
Because `char` may suggest a meaning of the data, which isn't there. How about an `std::shared_ptr<std::byte>`?

Reply:
Sure.